### ISSUE TITLE
lglcassume isn't implemented in the library interface

### ISSUE THREAD

**Issue by XeroOl** [arminbiere/lingeling#10]
The function `lglcassume` is declared in the header `lglib.h`, but doesn't actually do anything. It calls the empty function `lglecassume` which doesn't have any effects.
```c
void lglcassume (LGL * lgl, int elit);
```

**Comment by arminbiere** [comment]
Yep this is a stub which maybe at one point is implemented.  It corresponds to the CaDiCaL constrain API call.
